pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -768,6 +768,10 @@ func TestFitScore(t *testing.T) {
 
 			var gotPriorities framework.NodeScoreList
 			for _, n := range test.nodes {
```

Comment:
Please add the test cases to confirm if this plugin is working correctly if PreScore is NOT enabled.